jbpm-workitems/jbpm-workitems-core/src/main/java/org/jbpm/process/workitem/core/AbstractLogOrThrowWorkItemHandler.java

```diff
@@ -77,6 +80,13 @@ protected void handleException(Throwable cause,
         }
     }
 
+    private boolean canExceptionTriggerSubprocess(Throwable cause) {
+        while (cause != null && !(cause instanceof OptimisticLockException)) {
```

Comment:
Could it be more generic if the comparison is with PersistenceException?

Reply:
done